2010-05-24  Aurelien Jarno  <aurelien@aurel32.net>

	* sysdeps/wordsize-64/tst-writev.c: include <limits.h>. 

diff a/sysdeps/wordsize-64/tst-writev.c b/sysdeps/wordsize-64/tst-writev.c
---
 sysdeps/wordsize-64/tst-writev.c |    1 +
 1 file changed, 1 insertion(+)

--- a/sysdeps/wordsize-64/tst-writev.c
+++ b/sysdeps/wordsize-64/tst-writev.c
@@ -17,6 +17,7 @@
    <http://www.gnu.org/licenses/>.  */
 
 #include <fcntl.h>
+#include <limits.h>
 #include <paths.h>
 #include <stdio.h>
 #include <stdlib.h>
